index.bs

```diff
@@ -521,29 +521,19 @@ which would have issues of its own.
 The term <dfn id="concept-device-sensor">device sensor</dfn> refers to a device's underlying
 physical sensor instance.
 
-The [=device sensors=] measure different physical quantities
-and provide corresponding <dfn>raw sensor readings</dfn>
-which are a source of information about the user and their environment.
+A [=device sensor=] measures different physical quantities
```

Comment:
Before it was "sensors => physical quantities", if we change to singular, should it be "sensor => physical quantity"

Reply:
and why to emphasize `different physical quantities`?

Reply:
"different" could be dropped :)

Reply:
Fixed.